packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -23,7 +23,11 @@ TODO: Remove this section if there are not any general updates.
 
 ## Performance updates
 
-TODO: Remove this section if there are not any general updates.
+- Added a Queued Microtasks tab to the VM Tools screen, which allows a user to
```

Comment:
move this out of the Performance updates section and into the VM tools section

Reply:
Yeah go ahead and add a section "Advanced developer mode updates" and you can put this entry there. If you don't mind, can you add the same section to the bottom of the template? https://github.com/flutter/devtools/blob/master/packages/devtools_app/release_notes/helpers/release_notes_template.md/#L1

Reply:
Done.